static/app/views/issueDetails/groupRelatedIssues/index.spec.tsx

```diff
@@ -154,6 +154,12 @@ describe('Related Issues View', function () {
     expect(
       await screen.findByText('No trace-connected related issues were found.')
     ).toBeInTheDocument();
+    const linkButton = screen.getByRole('button', {name: /open in issues/i});
+    expect(linkButton).toHaveAttribute(
+      'href',
+      // Opening in Issues needs to include the group we are currently viewing
+      `/issues/?query=issue.id:[${groupId},${group1},${group2}]`
```

Comment:
The table displays groups 1 & 2, however, the "Open in Issues" button shows all groups.